spec/registry.md

```diff
@@ -186,3 +186,17 @@ which only expects the `accord` option:
 
     let $obj = {$object :noun case=nominative}
     {You see {$color :adjective article=indefinite accord=$obj} {$obj}!}
+
+## Built-in Registry
+
+As hinted in the example above, implementations may <!-- FIXME: may vs should --> provide built-in functions to their users in order to perform common operations in a standardized manner instead of having to rely on custom functions for everything. The working group agrees that these builtins provided by implementations should expose the following capabilities to their users:
```

Comment:
The default registry will contain function signatures that every implementation is **_required_** to provide. So this is neither MAY nor SHOULD: it is MUST.

Note that RFC2119 keywords MUST be capitalized.

You should think of this document as being part of the specification. This section shouldn't refer to "the working group" or what our thoughts are. Adopt the point of view of an implementer ("what do I have to do here?") or a user ("how do I use this?")

```suggestion
The **_<dfn>built-in registry</dfn>_** is the set of _functions_, together with
their options, that are present in every implementation.
Implementations MUST provide each of the built-in functions defined in the 
_built-in registry_, including all of the options.
They MAY include additional options, although implementers are cautioned to
avoid options which might conflict with future standardization.

These include the following capabilities:
```

Reply:
Please provide line breaks.